sunpy/net/attr.py

```diff
@@ -60,6 +113,55 @@ def collides(self, other):
     def __eq__(self, other):
         return dict(vars(self)) == dict(vars(other))
 
+    @classmethod
+    def update_values(cls, adict):
+        """
+        This is the method that clients will use to register their `Attrs`.
+        The input should be a dictionary.
+        Each key should be type(attrs.x) of any attrs require to exist for `_can_handle_query` to return successfully.
```

Comment:
I would split the bit about `_can_handle_query` out from the description of the dict. Maybe put it at the end like "it is recommended that clients register values for all attrs used by their `_can_handle_query` method.